en/cpp/quickstart.md

```diff
@@ -1,72 +1,52 @@
 # QuickStart
 
-This quickstart gets you started developing MAVSDK C++ core applications on macOS and Ubuntu Linux.
-It demonstrates how to run a simple MAVSDK example against a simulated vehicle, and observe the operation using the console and in a ground station.
+MAVSDK is essentially a C++ library (or rather multiple library files) that you can include and link in your C++ application.
```

Comment:
I prefer the original text - it says what this quickstart is going to show you how to do.

Reply:
Yes, I'll revisit that. It's just that I want to get away from SDK and core. It's marketing terms that others wanted and are actually incorrect and meaningless.

Reply:
Fair enough. Maybe do both.

----

This quickstart shows you how to build and run a simple MAVSDK C++ example application against a simulated vehicle, and observe the operation using the console and in a ground station (on both Ubuntu and macOS).

> **Note** The MAVSDK is essentially a C++ library (or rather multiple library files) that you can include and link in your C++ application.

Reply:
Perfect, thanks.